### PR TITLE
Add readme and contributions guides

### PR DESCRIPTION
**Pull request by dgutride**
These can be viewed more easily here:
https://github.com/dgutride/patternfly-ng/tree/readme

@catrobson

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+The following should be included in all PR's
+ - A clear commit message describing what has been done
+ - A reference to the [issue][github-issues] being addressed (whether it's a bug being fixed or a new component being added, a GitHub issue should be added to track it).
+ - A unit test covering the included behavior
+ - Documentation:
+  - Add a new examples folder within the component folder if it doesn't already exist
+  - Add an example component following the style of one of the existing components.  
+  - Examples should include a working sample including the ability to toggle configuration values to demonstrate the full functionality of a component
+  - Links to the example code and generated typedoc documenation should be included following the example below
+ ``` angular2html
```
Comment (AdamJ) [inline CONTRIBUTING.md]: this doesn't appear to be recognized by the prettifyer - can it be changed to just ```html ?

**Review by gperdomor — APPROVED** [review]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+## <a name="issue"></a> Found a bug or issue?
+Please help us out if you find a bug by submitting an issue to our [GitHub Repository][github-issues].
+We also welcome [PR's][github-pr] if you'd like to submit a fix.
+
+## <a name="submit"></a> Submission guidelines
+The following should be included in all PR's
+ - A clear commit message describing what has been done
+ - A reference to the [issue][github-issues] being addressed (whether it's a bug being fixed or a new component being added, a GitHub issue should be added to track it).
+ - A unit test covering the included behavior
+ - Documentation:
```
Comment (catrobson) [inline CONTRIBUTING.md]: Are the next bullets meant to be indented, or is this bullet missing information?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+ - Search our [GitHub issues][github-issues]
+ - Join our patternfly-ng channel on [Slack](http://slack.patternfly.org)
+ - Join our mailing-list following the instructions on [patternfly.org](http://www.patternfly.org/community/)
+
+
+## <a name="issue"></a> Found a bug or issue?
+Please help us out if you find a bug by submitting an issue to our [GitHub Repository][github-issues].
+We also welcome [PR's][github-pr] if you'd like to submit a fix.
+
+## <a name="submit"></a> Submission guidelines
```
Comment (catrobson) [inline CONTRIBUTING.md:28]: I'm not sure whether this should be part of submission guidelines or another section about Submission review process, but I think we should explain that submissions will be reviewed against PatternFly standards.  The text I've written up about this:

Any Angular-ng patterns submitted for contribution will be reviewed by designers on this repo associated with the PatternFly project to ensure that contributions match the PatternFly design standards for each component.  This design review is used to make sure that it meets one of the following criteria:

- Matches the "gold standard" design specifications as designed by PatternFly
- Matches the design specifications provided for a certain project and has been made general enough to work for other projects
- Goes through a design review and subsequent matching design documentation issues are created for PatternFly to match this contribution

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+     </tab>
+     <tab heading="typescript">
+       <include-content src="/src/app/list-view/examples/list-view-basic-example.component.ts"></include-content>
+     </tab>
+   </tabset>
+ </div>
+ ```
+
+## <a name="rules"></a> Our coding guidelines
+We follow the [Angular style guide](https://angular.io/guide/styleguide) for all Angular code.  Most general questions about how things should be named, where files should be lived and overall file structure can be answered by reviewing those guidelines.
```
Comment (catrobson) [inline CONTRIBUTING.md]: "where files should live"

File: README.md [hunk 5]
```diff
@@ -1,2 +1,77 @@
...
+  [header]="'test header'"
+  [message]="'this is a notification'"
+  [showClose]="'true'"
+  [type]="'success'">
+</pfng-toast-notification>
+```
+
+## <a name="question"></a> Do you have a question?
+There are a few things that can be done to resolve any questions or problems you might have.
+ - Search our [GitHub issues][github-issues]
```
Comment (catrobson) [inline README.md:73]: Not a live link by the looks of it.

**Review by catrobson — CHANGES_REQUESTED** [review]
Left a few comments and suggestions.

**Review by dlabrecq — APPROVED** [review]
LGTM

**Review by AdamJ — APPROVED** [review]

**Comment by dgutride** [comment]
thanks @catrobson - all changes have been made

**Review by gperdomor — APPROVED** [review]

**Pull request merged**